src/NUnitFramework/tests/Constraints/UniqueItemsConstraintTests.cs

```diff
@@ -41,13 +43,19 @@ public void SetUp()
         }
 
         static object[] SuccessData = new object[] { new int[] { 1, 3, 17, -2, 34 }, new object[0] };
-        static object[] FailureData = new object[] { new object[] { new int[] { 1, 3, 17, 3, 34 }, "< 1, 3, 17, 3, 34 >" } };
+        static object[] FailureData = new object[] { new object[] {
+            new int[] { 1, 3, 17, 3, 34 }, 
+            "< 1, 3, 17, 3, 34 >" + Environment.NewLine + "  Not unique items: < 3 >" }
+        };
 
-        [Test]
+        [Test, SetCulture("en-US")]
```

Comment:
As noted in https://github.com/nunit/nunit/pull/3502#discussion_r405144602
The class has been updated to use a `IEqualityComparer<T>`:
https://github.com/nunit/nunit/pull/3523/files#diff-f6b841c6c7eed9838e33cf9082f26944R147

Which caused some funny behaviour on Azure Linux (which uses the `en-POSIX-us` culture). See this for more info: https://github.com/nunit/nunit/pull/3416

Reply:
I think `SetCulture("")` sets it to invariant, but I forget.

Reply:
I just tested this, and it works. Thanks!